mitosheet/mitosheet/public/v3/sheet_functions/number_functions.py

```diff
@@ -57,62 +58,65 @@ def AVG(*argv: Union[int, float, None, pd.Series, RollingRange]) -> NumberFuncti
 @cast_values_in_arg_to_type('number')
 def MAX(*argv: Union[int, float, None, pd.Series, RollingRange]) -> NumberFunctionReturnType:
     
-    result: Union[pd.Series, float, int] = -sys.maxsize - 1
-
-    def get_new_result(_result: Union[pd.Series, float, int], new_value: Union[pd.Series, float, int]) -> Union[pd.Series, float, int]:
-        if isinstance(_result, pd.Series):
-            if isinstance(new_value, pd.Series):
-                return pd.concat([_result, new_value], axis=1).max(axis=1)
-            else:
-                return _result.apply(lambda v: np.nanmax([v, new_value]))
-        else:
-            if isinstance(new_value, pd.Series):
-                return new_value.apply(lambda v: np.nanmax([v, _result]))
-            else:
-                return max(_result, new_value)
+    result = get_final_result_series_or_primitive(
+        -sys.maxsize - 1,
+        argv,
+        lambda df: df.sum().sum(),
+        lambda previous_value, new_value: max(previous_value, new_value),
+        lambda previous_series, new_series: pd.concat([previous_series, new_series], axis=1).max(axis=1)
+    )
 
-    for arg in argv:
-        if isinstance(arg, pd.DataFrame):
-            df_sum = arg.sum().sum()
-            result = get_new_result(result, df_sum)
+    # If we don't find any arguements, we default to 0 -- like Excel
+    kept_default_max_value = not isinstance(result, pd.Series) and result == (-sys.maxsize - 1)
+    return result if not kept_default_max_value else 0
 
-        elif isinstance(arg, RollingRange):
-            new_series = arg.apply(lambda df: df.sum().sum())
-            result = get_new_result(result, new_series)
-            
-        elif isinstance(arg, pd.Series):
-            result = get_new_result(result, arg)
-            
-        elif arg is not None:
-            result = get_new_result(result, arg)
+
+@cast_values_in_arg_to_type('number')
+def MIN(*argv: Union[int, float, None, pd.Series, RollingRange]) -> NumberFunctionReturnType:
+
+    result = get_final_result_series_or_primitive(
+        sys.maxsize,
```

Comment:
Not sure if it matters, but for the MAX function we used `-sys.maxsize - 1`. Do we want to use `sys.maxsize - 1` here?

Reply:
Maybe we could create a MAX_SIZE constant that we use in all these cases. Sometimes -MAX_SIZE, other times MAX_SIZE

Reply:
You do minus one because that's what the maximum negative value is (they aren't symmetric). So defining a single variable wouldn't work in this case.